installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc

```diff
@@ -15,17 +15,24 @@ toc::[]
 [id="requirements-for-installing-ocp-on-ash"]
 == Requirements for installing {product-title} on Azure Stack Hub
 
-Before installing {product-title} on Microsoft Azure Stack Hub, you must configure an Azure account. See xref:../../installing/installing_azure_stack_hub/installing-azure-stack-hub-account.adoc#installing-azure-stack-hub-account[Configuring an Azure Stack Hub account] for details about account configuration, account limits, DNS zone configuration, required roles, and creating service principals.
+Before installing {product-title} on Microsoft Azure Stack Hub, you must configure an Azure account.
 
-You must manually manage your cloud credentials when installing a cluster to Azure Stack Hub. Do this by configuring the Cloud Credential Operator (CCO) for manual mode before you install the cluster. For more information, see xref:../../installing/installing_azure/manually-creating-iam-azure.adoc#manually-creating-iam-azure[Manually creating IAM for Azure].
+See xref:../../installing/installing_azure_stack_hub/installing-azure-stack-hub-account.adoc#installing-azure-stack-hub-account[Configuring an Azure Stack Hub account] for details about account configuration, account limits, DNS zone configuration, required roles, and creating service principals.
 
 [id="choosing-a-method-to-install-ocp-on-ash"]
 == Choosing a method to install {product-title} on Azure Stack Hub
 
-You can install {product-title} on Azure Stack Hub using user-provisioned infrastructure. This means you must manage and maintain the cluster resources yourself. Installing {product-title} on Azure Stack Hub using an installation program that automatically provisions the cluster infrastructure is not supported at this time.
+You can install {product-title} on installer-provisioned or user-provisioned infrastructure. The default installation type uses installer-provisioned infrastructure, where the installation program provisions the underlying infrastructure for the cluster. You can also install {product-title} on infrastructure that you provision. If you do not use infrastructure that the installation program provisions, you must manage and maintain the cluster resources yourself.
 
 See xref:../../architecture/architecture-installation.adoc#installation-process_architecture-installation[Installation process] for more information about installer-provisioned and user-provisioned installation processes.
 
+[id="choosing-a-method-to-install-ocp-on-ash-installer-provisioned"]
+=== Installing a cluster on installer-provisioned infrastructure
+
+You can install a cluster on Azure Stack Hub infrastructure that is provisioned by the {product-title} installation program, by using one of the following methods:
```

Comment:
```suggestion
You can install a cluster on Azure Stack Hub infrastructure that is provisioned by the {product-title} installation program, by using the following method:
```

Reply:
You got the s/methods/method, but still says "one of"

Reply:
Ugg. Thank you.